docs/topics/native/native-debugging.md

```diff
@@ -213,9 +238,55 @@ Process 80496 launched: './program.kexe' (x86_64)
 (lldb) expression -- (int32_t)Konan_DebugPrint(a_variable)
 (a_variable) one is 1(int32_t) $0 = 0
 (lldb)
+```
+
+## Debug iOS applications
+
+Debugging iOS application sometimes involves analyzing crash reports in detail.
+Crash reports typically require symbolication, the process of translating memory addresses into readable source code locations.
+
+To symbolicate addresses in Kotlin code (for example, for stack trace elements corresponding to Kotlin code), you need a
+special debug symbol (`.dSYM`) file.
```

Comment:
Minor: `.dSYM` files are used only for Apple platforms.
On other platforms, enabling the flag just adds debug information into the produced binaries (increasing their size in particular).